src/yoroi-wallets/cardano/nfts.ts

```diff
@@ -0,0 +1,25 @@
+import {getAssetFingerprint} from '../../legacy/format'
+import {AssetMetadata, NFTMetadata, YoroiNFT} from '../types'
+import {asciiToHex} from '../utils/parsing'
+
+export const convertNft = (assetMetadata: AssetMetadata, storageUrl: string): YoroiNFT => {
+  const policyId = Object.keys(assetMetadata)[0]
+  const assetNameKey = Object.keys(assetMetadata[policyId])[0]
+  const metadata: NFTMetadata = assetMetadata[policyId][assetNameKey]
+  const assetNameHex = asciiToHex(assetNameKey)
+  const fingerprint = getAssetFingerprint(policyId, assetNameHex)
+  const description = Array.isArray(metadata.description) ? metadata.description.join(' ') : metadata.description
+
+  return {
+    id: `${policyId}.${assetNameHex}`,
```

Comment:
does it make sense for this to be the asset fingerprint instead?

Reply:
yeah, it looks like asset explorers use asset fingerprint instead of `policyId.assetNameHex`